bedrock/base/templates/base-error.html

```diff
@@ -12,8 +12,10 @@
 {% block site_header %}
   <header class="mzp-l-content header-spacing">
     <div class="header-image">
-    {{ high_res_img('img/co-brand/firefox_moz_co-brand_sm.png', {'alt': 'Firefox, Mozilla'}) }}
+    <a href="https://www.firefox.com">{{ high_res_img('img/logos/firefox/logo-reality.png', {'alt': 'Firefox'}) }}</a>
```

Comment:
This is the logo for Firefox Reality browser, which is incorrect as it's not the regular Firefox logo. Also, a high-res version of this logo doesn't exist, so it's currently 404 in the page.

I'd suggest using `protocol/img/logos/firefox/logo-word-hor-xs.png`

The `href` should use our URL helper:

```
href="{{ url('firefox') }}"
```